File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function solution(binomial) {
    const s = binomial.split(" ");
    const a = parseInt(s[0]);
    const b = parseInt(s[2]);

    switch(s[1]) {
        case "+" :
            return a + b;
        case "-" :
            return a - b;
        case "*" :
            return a * b;
    }
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    s = binomial.split()
    a = int(s[0])
    op = s[1]
    b = int(s[2])

    if op == "+":
        return a + b
    elif op == "-":
        return a - b
    elif op == "*":
        return a * b